packages/markup-frontstage/README.md

```diff
@@ -0,0 +1,52 @@
+# @bentley\markup-frontstage-react
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+The markup-frontstage-react provides Markup Frontstage for adding, editing, deleting markup elements in iModel.js application.
```

Comment:
Missing a determiner (I think that's what they're called) here, I'd go with something like "provides a Markup Frontstage" or "provides the Markup Frontstage".